monkestation/code/game/objects/items/plushes.dm

```diff
@@ -40,3 +40,63 @@
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('monkestation/sound/voice/laugh/moth/mothlaugh.ogg'=1)
 	has_creepy_icons = TRUE
+
+
+/obj/item/toy/plush/goatplushie
+	name = "strange goat plushie"
+	icon = 'monkestation/icons/obj/plushes.dmi'
+	icon_state = "goat"
+	desc = "Despite its cuddly appearance and plush nature, it will beat you up all the same. Goats never change."
+	squeak_override = list('sound/weapons/punch1.ogg'=1)
+	/// Whether or not this goat is currently taking in a monsterous doink
+	var/going_hard = FALSE
+	/// Whether or not this goat has been flattened like a funny pancake
+	var/splat = FALSE
+
+/obj/item/toy/plush/goatplushie/Initialize(mapload)
+	. = ..()
+	var/static/list/loc_connections = list(
+		COMSIG_TURF_INDUSTRIAL_LIFT_ENTER = .proc/splat,
```

Comment:
Does this actually exist anywhere?

Reply:
No it doesn't changing that to a unique proc now